.kokoro/tests/run_tests.sh

```diff
@@ -96,12 +96,6 @@ fi
 source ./testing/test-env.sh
 export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/testing/service-account.json
 
-# Import secrets for AWS integration testing. This can be used for products
```

Comment:
We want to use secret store instead.

Undoes this change in this commit: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/7091/files#diff-ff2ac0c2d9eac8a4fc58f24e20895504b9c25a1b6d6c1f6f55a5978c1f5aefc3